package.json

```diff
@@ -25,6 +25,8 @@
     "babel-plugin-transform-object-rest-spread": "^6.22.0",
     "babel-preset-es2015": "^6.22.0",
     "eslint": "^3.15.0",
+    "eslint-config-google": "^0.7.0",
+    "eslint-plugin-sort-requires": "^2.1.0",
```

Comment:
Please leave this out.  I'm open to changing lint set up for this repo but we should discuss it first and do it in a separate PR.